tools/report-converter/tests/unit/test_coccinelle_parser.py

```diff
@@ -0,0 +1,79 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+
+"""
+This module tests the correctness of the CoccinelleAnalyzerResult, which
+used in sequence transform Pyflakes output to a plist file.
```

Comment:
```suggestion
used in sequence transform Coccinelle output to a plist file.
```

Reply:
Oops! Lemme correct this and rebase it with the test commit.